java/code/src/com/redhat/rhn/manager/org/MigrationManager.java

```diff
@@ -130,11 +133,10 @@ public static void removeOrgRelationships(User user, Server server) {
         SystemManager.unsubscribeServerFromChannel(server, server.getBaseChannel());
 
         // Remove from all system groups:
-        ServerGroupManager manager = ServerGroupManager.getInstance();
         for (ManagedServerGroup group : server.getManagedGroups()) {
             List<Server> tempList = new LinkedList<Server>();
             tempList.add(server);
-            manager.removeServers(group, tempList);
+            SERVER_GROUP_MANAGER.removeServers(group, tempList);
```

Comment:
Nitpick: If we are using it only once, maybe inline it and directly use `GlobalInstanceHolder.SERVER_GROUP_MANAGER`

Reply:
I did this as an in between step since those classes should also be refactored at some point. The other reason was to keep references to GlobalInstanceHolder.* to a minimum so the find usage dialog in intellij is easier to work with.